tg/msg.py

```diff
@@ -125,3 +125,35 @@ def is_text(self):
     def is_downloaded(self):
         doc = self.get_doc(self.msg)
         return doc["local"]["is_downloading_completed"]
+
+    @property
+    def is_listened(self):
```

Comment:
I think this method should return `bool`

Reply:
Okay